compiler-core/src/analyse.rs

```diff
@@ -839,8 +839,14 @@ impl<'a, A> ModuleAnalyzer<'a, A> {
             for (i, RecordConstructorArg { label, ast, .. }) in
                 constructor.arguments.iter().enumerate()
             {
-                // Build a type from the annotation AST
-                let t = hydrator.type_from_ast(ast, environment)?;
+                // Build a type from the annotation AST// Build a type from the annotation AST
```

Comment:
Funny comment here

Reply:
Oops. must have accidentally copy-pasted.

Reply:
I've now fixed that comment.